e2e/cypress/integration/enterprise/system_console/team_scheme_permission_spec.js

```diff
@@ -117,62 +134,115 @@ const resetPermissionsToDefault = () => {
     cy.get('#confirmModalButton').click();
 
     // # Save
-    saveConfig();
+    cy.get('#saveSetting').click();
+    cy.waitUntil(() => cy.get('#saveSetting').then((el) => {
+        return el[0].innerText === 'Save';
+    }));
 };
 
-describe('System Scheme Channel Mentions Permissions Test', () => {
+describe('Team Scheme Channel Mentions Permissions Test', () => {
     before(() => {
         // * Check if server has license
         cy.requireLicense();
 
         // Reset permissions in system scheme to defaults.
         resetPermissionsToDefault();
 
+        // Delete any existing team override schemes
         deleteExistingTeamOverrideSchemes();
+
+        // # Visit the permissions page
+        cy.visit('/admin_console/user_management/permissions');
     });
 
```

Comment:
Similarly in this test as well, can you please add an after method to resetPermissionsToDefault()